### DISCUSSION TITLE
fan speed

### DISCUSSION THREAD

**Discussion by marcinion** [seerge/g-helper#1305] [Ideas]
Hi

Could you please consider to add to your amassing soft one option - direct selection of the fan speed ?
<img width="149" alt="Screenshot 2023-09-12 153142" src="https://github.com/seerge/g-helper/assets/144902392/f3662259-d0ac-42b6-b7da-270e6e820d27">

**Comment by seerge** [comment]
@marcinion hello, I didn't get your question.

You can already select any mode with appropriate fan curves in the app.

**Reply by marcinion** [reply]
Yes and No
I do not want to change CPU or GPU, just the fans speed



<img width="399" alt="image" src="https://github.com/seerge/g-helper/assets/144902392/93b80618-da9c-4b70-80fa-45ff605adf8f">

**Reply by seerge** [reply]
The screenshot you have posted from Atrofac also changes modes (i.e. whole package including power / etc).

Those modes are hardcoded in BIOS.

**Comment by marcinion** [comment]
OK, I did not know it - are you sure?

![image](https://github.com/seerge/g-helper/assets/144902392/9cc6c431-5535-450a-b056-b9f97778f22c)

They have more options like for silent

**Reply by seerge** [reply]
You can copy silent proifle 3 times (with + button under fans+power) and set custom fan curves for each version of silent.

**Comment by marcinion** [comment]
You are correct, so could we add more customisable buttons on the top?

**Reply by seerge** [reply]
@marcinion you can cycle between all modes (including "custom")  via FN+F5 / FN+Shift+F5 (backwards) and also select  them in a dropdown in fans+power, that should be enough.

I don't plan to overload UI of main screen with more buttons for now :)
